pycti/utils/opencti_stix2.py

```diff
@@ -768,7 +768,7 @@ def import_object(
             "channel": self.opencti.channel.import_from_stix2,
             "event": self.opencti.event.import_from_stix2,
             "note": self.opencti.note.import_from_stix2,
-            "observed-data": self.opencti.observed_data.import_from_stix2,
+            "observedData": self.opencti.observed_data.import_from_stix2,
```

Comment:
Are we sure about this ?
it seems anormal to don't follow the nomenclature here

Reply:
I tested the behavior... with observed-data, nothing is exported, with observedData the behavior is the one expected... maybe we should change the typo in the front... the correct name would be: 'observed-Data'

Reply:
fixed (with observed-data instead of observedData)